src/cli/cluster/cluster_manager.js

```diff
@@ -116,6 +116,7 @@ export default class ClusterManager {
               resolve(path, 'target'),
               resolve(path, 'scripts'),
               resolve(path, 'docs'),
+              resolve(path, 'legacy/plugins/siem/cypress'),
```

Comment:
prevents the Kibana server from restarting when files (tests) in this directory are saved in an editor